docs/src/content/docs/patterns/effect.md

```diff
@@ -28,4 +28,226 @@ import { Schema } from 'effect'
 ## `Equal` and `Hash` Traits
 
 LiveStore's reactive primitives (`LiveQueryDef` and `SignalDef`) implement Effect's `Equal` and `Hash` traits, enabling efficient integration with Effect's data structures and collections.
+
+## Effect Atom Integration
+
+LiveStore integrates seamlessly with [Effect Atom](https://github.com/effect-atom/effect-atom) for reactive state management in React applications. This provides a powerful combination of Effect's functional programming capabilities with LiveStore's event sourcing and CQRS patterns.
+
+Effect Atom is an external package developed by [Tim Smart](https://github.com/tim-smart) that provides a more Effect-idiomatic alternative to the `@livestore/react` package. While `@livestore/react` offers a straightforward React integration, Effect Atom leverages Effect API/patterns throughout, making it a natural choice for applications already using Effect.
+
+### Installation
+
+```bash
+pnpm install @effect-atom/atom-livestore @effect-atom/atom-react
+```
+
+### Store Creation
+
+Create a LiveStore-backed atom store with persistence and worker support:
+
+```ts
+// atoms.ts
+import { schema } from './schema'
+import { makePersistedAdapter } from '@livestore/adapter-web'
+import { AtomLivestore } from '@effect-atom/atom-livestore'
+import { unstable_batchedUpdates } from 'react-dom'
+
+// Create a persistent adapter with OPFS storage
+const adapter = makePersistedAdapter({
+  storage: { type: 'opfs' },
+  worker: LiveStoreWorker,
+  sharedWorker: LiveStoreSharedWorker,
+})
+
+// Create store atoms
+export const {
+  runtimeAtom,      // Access to Effect runtime
+  commitAtom,       // Commit events to the store
+  storeAtom,        // Access store with Effect
+  storeAtomUnsafe,  // Direct store access (may be undefined)
```

Comment:
The unsafe APIs are when you can assume the store is already loaded. So they can be used synchronously.